nengo_loihi/builder.py

```diff
@@ -27,9 +27,9 @@
     CxModel,
     CxProbe,
     CxSpikeInput,
-    CxSynapses,
 )
 from nengo_loihi.splitter import ChipReceiveNeurons, ChipReceiveNode
+from nengo_loihi.synapses import CxSynapses
```

Comment:
I'm wondering if we should have a `model` folder, which will hold all the formerly `Cx` object files, such as this one. Just to make it clear that they all go together to describe this level of model.